index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Valentine Jeremiah</h1>
+      <h3 class="sub-title">The Best Microverse pair programers</h3>
+      <figure id="img-div">
+        <img src="./images/Picture1-1.png" alt="" id="image">
+        <h1 id="img-caption">This is Jeremiah and Valentine</h1>

```

Comment:
You should consider using `h2` or `h3` for this, You should have only one `h1 tag` in your code for better `SEO` and semantics